CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [8.0.0](https://github.com/camptocamp/devops-stack-module-loki-stack/compare/v7.2.0...v8.0.0) (2024-04-23)
+
+
+### ⚠ BREAKING CHANGES
+
+* **eks:** add option to create IAM role for the metrics storage
+  * This is a breaking change because the attributes of the `logs_storage` variable has changed. Please check the README.adoc to check the differences.
```

Comment:
```suggestion
  * This is a breaking change because the attributes of the `logs_storage` variable has changed. Please check the README.adoc to see the differences.
```